### DISCUSSION TITLE
training topaz demonising model with frames collected in .eer format

### DISCUSSION THREAD

**Discussion by samuelhaysom** [tbepler/topaz#126] [Q&A]
This is more of a question than an issue but I was wondering if the latest version (0.2.5) can handle frame data in .eer format when training a demonising model, and then use this model to denoise .mrc files created from those .eer files using MotionCorr in RELION4? Or would I have to convert the frame data to .mrc files first to train the model?

**Comment by tbepler** [comment]
Topaz cannot read EER data at this point. If you want to train specialized denoising models, I recommend motion correcting the micrographs before training anyway. Have you tried the pre-trained model? Most likely you don't need to train a specialized one.

**Reply by samuelhaysom** [reply]
I've tried the pre-trained model on my data and wasn't 100% happy with my results. I'm looking at proteins in a liposomes membrane and rather than making them more obvious the demonising seems to have blurred the locations in the membrane where I think the protein is located. I wanted to test whether a custom model would improve things/give a different result. My reasoning is that the datasets used to train the demonising probably didn't include many proteoliposome datasets and so topaz might not handle such micrographs as well.

**Reply by tbepler** [reply]
Makes sense. Training on the motion corrected split frames is the way to go then.
